src/index.html

```diff
@@ -1,13 +1,333 @@
+                    <p class="about__work--time">Closed</p>
+                  </div>
+    
+                  <div class="about__work--section">
+                    <p class="about__work--day">Thu - Tue</p>
+                    <p class="about__work--day">Wen</p>
+                  </div>
+                </section>
+  
+                <p class="about__close">Closed on</p>
+                <p class="about__close-text">Thanksgiving Day, Dec 25, Jan 1</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section gallery" id="now-on-view">
+        <div class="container">
+          <h2 class="page__section-title gallery__title">Now On View</h2>
+
+          <p class="gallery__subtitle">Explore exhibition gallery guides, videos, articles, and more.</p>
+
+          <div class="gallery__block">
+            <div class="grid">
+              <div class="grid__item grid__item--desktop-1-8 grid__item--table-1-4">
+                <div class="gallery__photo gallery__photo--1"></div>
+              </div>
+                
+              <div class="grid__item grid__item--desktop-9-12 grid__item--table-5-6">
+                <div class="gallery__photo gallery__photo--2"></div>
+              </div>
+            </div>
+
+            <p class="gallery__text">A New Look at Old Masters</p>
+            <p class="gallery__date">December 12, 2020 – Through Spring 2023</p>
+          </div>
+
+          <div class="gallery__block">
+            <div class="gallery__block--flex">
+              <div class="gallery__photo gallery__photo--3"></div>
+              <div class="gallery__photo gallery__photo--4"></div>
+              <div class="gallery__photo gallery__photo--5"></div>
+            </div>
+
+            <p class="gallery__text">Bodhisattvas of Wisdom, Compassion, and Power</p>
+            <p class="gallery__date">March 27, 2021 – October 30, 2022</p>
+          </div>
+
+          <div class="gallery__block">
+            <div class="grid">
+              <div class="grid__item grid__item--desktop-1-4 grid__item--table-1-2">
+                <div class="gallery__photo gallery__photo--6"></div>
+              </div>
+                
+              <div class="grid__item grid__item--desktop-5-12 grid__item--table-3-6">
+                <div class="gallery__photo gallery__photo--7"></div>
+              </div>
+            </div>
+
+            <p class="gallery__text">Inspiring Walt Disney: The Animation of French Decorative Arts</p>
+            <p class="gallery__date">December 10, 2021 – March 6, 2022</p>
+          </div>
+
+          <div class="grid grid--table grid--desktop">
+            <div class="grid__item grid__item--desktop-5-8 grid__item--table-2-5">
+              <a href="#" class="button">
+                View all exhibitions
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
```

Comment:
Gallery is content block, all images should be added using <img /> for example but definitely not as background image